src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -11,18 +13,25 @@ class SubjectHeading extends React.Component {
     const {
       subjectHeading,
       container,
+      sortBy,
     } = this.props;
     const {
       children,
+      range,
     } = subjectHeading;
     this.state = {
       open: !!children,
       narrower: (children || []),
+      sortBy: sortBy || "alphabetical",
+      range: range || new Range(0, Infinity, [{ start: 0, end: Infinity }]),
```

Comment:
Could that be the default Range or a method in the Range class?

Reply:
good idea